kms/api-client/quickstart.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-# Copyright 2017 Google, Inc
+# Copyright 2020 Google LLC
```

Comment:
Here and there
Please don't update the copyright year. 
I think it is ok to keep the copyright declaration.
If you really want to add Google LLC bit, you can do it, but you have to keep the original year and the original name.

Reply:
I'm not a lawyer, so If you unsure, consult OSPO

Reply:
Done